VCFv4.3.tex

```diff
@@ -547,6 +594,8 @@ \section{INFO keys used for structural variants}
 
 \footnotesize
 \begin{verbatim}
+##INFO=<ID=TDLOC,Number=1,Type=String,Description="Position and orientation of tandem duplication; must be one of 
+       (Direct, Inverted-downstream, or Inverted-upstream)">
```

Comment:
I was not able to keep this to one line, nor to wrap it, with the current text. I will try working up an alternative that fits on one line.